lib/bot-service.js

```diff
@@ -70,9 +85,11 @@ function decodeToken (token) {
         if (decoded.hasOwnProperty('bot')) {
             return decoded.bot;
         }
-        logger.error('Invalid Response. Missing key `bot` in response JSON');
+      logger.error('Invalid Response. Missing key `bot` in response JSON');
+      return new Error('Invalid Response. Missing key `bot` in response JSON');
```

Comment:
ah now I understand why you added the error check. This should just throw the error instead of returning it. I can explain it if you want, let me know.

Reply:
Changed to throw error